xml/docu_styleguide.taglist.xml

```diff
@@ -0,0 +1,1175 @@
+       </screen>
+      </entry>
+      <entry>
+       A character or string indicating the start of an input field in a
+       computer display
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+quote
+       </screen>
+      </entry>
+      <entry>
+       To quote from sources, such as books. In all other cases, do not use
+       quotation marks.
+      </entry>
+      <entry>Yes</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+replaceable
+       </screen>
+      </entry>
+      <entry>
+       Content that may or must be replaced by the user
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+systemitem
+       </screen>
+      </entry>
+      <entry>
+       A system-related item or term
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+subscript
+       </screen>
+      </entry>
+      <entry>
+       <para>A subscript. For example:</para>
+       <para>H<subscript>2</subscript>O</para>
+      </entry>
+      <entry>Yes</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+superscript
+       </screen>
+      </entry>
+      <entry>
+       <para>A superscript. For example:</para>
+       <para>X<superscript>2</superscript></para>
+      </entry>
+      <entry>Yes</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+tag
+       </screen>
+      </entry>
+      <entry>
+       A component of XML (or SGML) markup
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+trademark
+       </screen>
+      </entry>
+      <entry>
+       To mark a trademark with <trademark>TM</trademark>
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+uri
+       </screen>
+      </entry>
+      <entry>
+       For links that should not be clickable
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+varname
+       </screen>
+      </entry>
+      <entry>
+       The name of a variable
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+xi:include
+       </screen>
+      </entry>
+      <entry>
+       To construct documents from different files
+      </entry>
+      <entry>No</entry>
+     </row>
+     <row>
+      <entry>
+       <screen>
+xref
+       </screen>
+      </entry>
+      <entry>
+       A cross-reference to another part of the document
+      </entry>
+      <entry>No</entry>
+     </row>
+    </tbody>
+   </tgroup>
+  </informaltable>
+ </sect3>
+</sect2>
+ 
+<sect2 xml:id="sec-taglist-ITS">
+ <title>Using ITS tags</title>
+ <para>
+  With Geekodoc 2 it is possible to use the Internationalization Tag Set (ITS)
+  tags to explicit indicate that a tag should be translated or not.
+ </para>
+ <para>
+  For this you need to have the its namespace enabled in the XML file. Make
```

Comment:
I think, it's better to be consistent and use the already introduced ITS acronym:

```suggestion
  For this you need to have the ITS namespace enabled in the XML file. Make
```